Week02/lesson04/homework/relaxr_starter_code/index.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="utf-8">
+	<meta name="description" content="Relaxr, the perfect personal assistant. Get Piece of mind with a single tap.">
+	<meta name="keywords" content="Relaxr, Personal Assistant, PA, AI Technology">
+	<title>Relaxr | The Perfect PA</title>
+
+	<!-- external CSS links -->
+	<link rel="stylesheet" href="css/reset.css">
+	<link rel="stylesheet" href="css/style.css">
+	<link href="https://fonts.googleapis.com/css?family=Open+Sans:400,300,300italic,600,800" rel="stylesheet" type="text/css">
+</head>
+
+<body>
+
+	<div id="container">
+		<header>
+			<h1>Relaxr</h1>
+			<h2>Get piece of mind with a single tap</h2>
+			<a href="http://www.relaxr.com/signup"><div id="button1">Get it Now</div></a>
+		</header>
+
+		<main>
+			<h3>Benefits</h3>
+			<p>The perfect personal assistant. Relaxr does your job for you so you can enjoy life as it is meant to be.</p>
+				<ul>
+					<li>Schedule meetings for you</li>
```

Comment:
I can see when I open your code in Atom that your indentation is fine, but it seems to be getting messed up in Github.

In Atom, go to `Atom > Preferences` and then there, within `Settings`, make sure you've got `Soft Tabs` ticked, the `Tab Length` black (so that it picks up the default of 2) and `Tab Type` set to `Auto`. Like this:

![image](https://cloud.githubusercontent.com/assets/5670279/17462186/a0ec0b40-5c9c-11e6-899e-0b3261dff52d.png)